src/tests/ftest/daos_test/suite.yaml

```diff
@@ -23,7 +23,7 @@ timeouts:
   test_daos_epoch_recovery: 104
   test_daos_md_replication: 104
   test_daos_rebuild_simple: 1800
-  test_daos_drain_simple: 2000
+  test_daos_drain_simple: 3600
```

Comment:
Does this really need another 26 minutes?

Reply:
yeah, sometimes, drain test might take longer time in 2.4, since adding a few new test cases and also pool map sync might take longer time during drain test.

Reply:
Curious - Are there plans to investigate ways to have tighter bounds for operations like drain, rebuild, etc?